tests/unit/test_channels/test_plugin.py

```diff
@@ -77,7 +77,6 @@ def test_plugin_dependency_signature_namespace(memory_backend: MemoryChannelsBac
     assert app.signature_namespace["ChannelsPlugin"] is ChannelsPlugin
 
 
-@pytest.mark.flaky(reruns=5)
```

Comment:
I removed all the flaky instances : it will definitely play bad with unclosed ressources and makes it hard to track down things